tei_adapter/terraform/rds_id_extractor.tf

```diff
@@ -48,18 +48,19 @@ resource "aws_security_group" "database_sg" {
 
 module "tei_id_extractor_rds_cluster" {
   source             = "../../infrastructure/critical/modules/rds"
-  cluster_identifier = "tei-adapter-cluster-delta"
+  cluster_identifier = "tei-adapter-cluster"
   database_name      = "pathid"
   username           = local.rds_username
   password           = local.rds_password
 
   # This instance class provides us with 45 connections per instance.
   instance_count = 1
-  instance_class = "db.t3.medium"
+  instance_class = "db.t3.small"
```

Comment:
Is the instance class comment still correct? (And does it matter?)

Reply:
Looking [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Managing.Performance.html), the comment is actually correct, which means the comments in the id minter terraform are not